packages/components/src/custom-select-control-v2/test/index.tsx

```diff
@@ -51,12 +51,18 @@ const legacyProps = {
 
 const LegacyControlledCustomSelect = ( {
 	options,
+	onChange,
+	...restProps
 }: LegacyCustomSelectProps ) => {
 	const [ value, setValue ] = useState( options[ 0 ] );
 	return (
-		<CustomSelect
-			{ ...legacyProps }
-			onChange={ ( { selectedItem }: any ) => setValue( selectedItem ) }
+		<UncontrolledCustomSelect
+			{ ...restProps }
```

Comment:
Ensure that the props from each test instance are actually passed through, instead of just spreading the scaffolded `legacyProps`.